nginx/no_ssl.conf

```diff
@@ -0,0 +1,29 @@
+server {
+    server_name localhost;
+    listen 80;
+
+    location /static/ {
+        root /;
+    }
+
+    location /media/image_upload/ {
+        root /;
+    }
+
+    location / {
+        proxy_pass http://django:8080;
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection "Upgrade";
+    }
+}
+
+# Port 8080 is deprecated, redirect to 80
+server {
+    server_name localhost;
+    listen 8080;
+
+    location / {
+        return 301 http://$host:80$request_uri;
```

Comment:
I may again be mis-understanding something, but why are we redirecting to HTTP and not HTTPS?

Reply:
This is the config file to use when the SSL cert has not been provided, so only HTTP is possible (hence why I called it "no_ssl"). It does redirect to HTTPS in the default conf